web/src/app/shared/components/workbasket-filter/workbasket-filter.component.html

```diff
@@ -62,14 +62,15 @@
       <!-- TYPE FILTER -->
       <button mat-stroked-button [matMenuTriggerFor]="menu" matTooltip="Filter workbaskets by type">
         Filter by type
-        <mat-icon *ngIf="filter.type" style="color: #555">filter_list</mat-icon>
+        <mat-icon *ngIf="filter.type.length == 0" style="color: #555">filter_list</mat-icon>
         <taskana-administration-icon-type *ngIf="filter.type[0]"
                                           [type]="filter.type[0]"></taskana-administration-icon-type>
       </button>
 
       <mat-menu #menu="matMenu">
         <button mat-menu-item *ngFor="let type of allTypes | mapValues" (click)="selectType(type.key); search()">
-          <taskana-administration-icon-type [type]='type.key' [text]="type.value"></taskana-administration-icon-type>
+          <span *ngIf="type.value === 'All'"> <mat-icon style="color: #555" class="filter__all-icon">filter_list</mat-icon> All </span>
+          <taskana-administration-icon-type *ngIf="type.value !== 'All'" [type]='type.key' [text]="type.value"></taskana-administration-icon-type>
         </button>
       </mat-menu>
 
```

Comment:
I would maybe change the name of the second class from filter to filter__expanded